lib/grape-swagger/doc_methods/move_params.rb

```diff
@@ -113,6 +118,17 @@ def build_definition(name, verb = nil)
           name
         end
 
+        def prepare_nested_types(params)
+          params.each do |param|
+            param.tap do |x|
```

Comment:
This seems silly, all this does is rename `param` into `x`, wjy don't you just do `do |x|` above or, better leave it as `param`?